vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/OidcSiopWallet.kt

```diff
@@ -180,13 +180,13 @@ class OidcSiopWallet(
         request: AuthenticationRequestParametersFrom
     ): KmmResult<AuthenticationResponseResult> = catching {
         val response = createAuthnResponseParams(request).getOrThrow()
-        if (request.parameters.responseType == null
-            || (!request.parameters.responseType!!.contains(ID_TOKEN)
-                    && !request.parameters.responseType!!.contains(VP_TOKEN))
-        ) {
-            Napier.w("createAuthnResponse: Unknown response_type ${request.parameters.responseType}")
-            throw OAuth2Exception(Errors.INVALID_REQUEST)
-        }
+        request.parameters.responseType?.let {
+            if (!it.contains(ID_TOKEN) && !it.contains(VP_TOKEN))
+                throw OAuth2Exception(Errors.INVALID_REQUEST)
+                    .also { Napier.w("createAuthnResponse: Unknown response_type $it") }
+        } ?: throw OAuth2Exception(Errors.INVALID_REQUEST)
+            .also { Napier.w("createAuthnResponse: response_type null in ${request.parameters}") }
+
```

Comment:
Reformatting once (with action "Reformat code") does not change anything. Only when reformatting again (when the popup "Second Reformat" asks for removing custom line breaks, it changes the code. But I'm never doing this, because it will mess up readability.